examples/runtime-sdk/rofl-oracle/src/main.rs

```diff
@@ -12,21 +12,24 @@ impl App for OracleApp {
 
     /// Identifier of the application (used for registrations).
     fn id() -> AppId {
-        "rofl1qp55evqls4qg6cjw5fnlv4al9ptc0fsakvxvd9uw".into() // TODO: Replace with your application ID.
+        "rofl1qqn9xndja7e2pnxhttktmecvwzz0yqwxsquqyxdf".into() // TODO: Replace with your application ID.
     }
 
     /// Return the consensus layer trust root for this runtime; if `None`, consensus layer integrity
-    /// verification will not be performed.
+    /// verification will not be performed (e.g. Localnet).
     fn consensus_trust_root() -> Option<TrustRoot> {
-        // The trust root below is for Sapphire Testnet at consensus height 22110615.
-        Some(TrustRoot {
-            height: 22110615,
-            hash: "95d1501f9cb88619050a5b422270929164ce739c5d803ed9500285b3b040985e".into(),
-            runtime_id: "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c".into(),
-            chain_context: "0b91b8e4e44b2003a7c5e23ddadb5e14ef5345c0ebcb3ddcae07fa2f244cab76"
-                .to_string(),
-        })
+        None
     }
+//     fn consensus_trust_root() -> Option<TrustRoot> {
+//         // The trust root below is for Sapphire Testnet at consensus height 22110615.
+//         Some(TrustRoot {
+//             height: 22110615,
+//             hash: "95d1501f9cb88619050a5b422270929164ce739c5d803ed9500285b3b040985e".into(),
+//             runtime_id: "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c".into(),
+//             chain_context: "0b91b8e4e44b2003a7c5e23ddadb5e14ef5345c0ebcb3ddcae07fa2f244cab76"
+//                 .to_string(),
+//         })
+//     }
```

Comment:
I am not convinced this should be the default as then people will omit the trust root.

Reply:
CLI defining the trust root would work best. Could Oasis CLI also verify that the ROFL .orc is ready to roll once it's compiled (e.g. check if the trust root is not None, which app ID is defined etc.)?

Reply:
Switched back.

Reply:
FYI opened https://github.com/oasisprotocol/cli/issues/291 to address this.